bin/replication.js

```diff
@@ -403,7 +429,7 @@ class _SetupReplication {
 
     setupReplication(cb) {
         return async.waterfall([
-            next => async.parallel({
```

Comment:
This change was intentional, the idea was that we want to stop immediately if something fails and it translates to a smaller headache while debugging.